src/Resources/js/modules/participation-pre-toggle-handler.ts

```diff
@@ -41,6 +56,22 @@ export class ParticipationPreToggleHandler {
             && 0 < $checkbox.closest('.meal').find('.variation-row .text-variation').length; // has variations
     }
 
+    private isCombinedMealParticipation($checkbox: JQuery): boolean {
+        return 0 < $checkbox.closest('.meal-row.combined-meal').length;
+    }
+
+    private getSimpleDishSlugs($mealContainer: JQuery): string[]
+    {
+        let dishes: string[] = [];
+        $mealContainer
+            .find('.meal-row:not(.combined-meal)')
```

Comment:
Instead of the combined class, we could also check for the data-combined attribute.